style.css

```diff
@@ -0,0 +1,550 @@
+@media screen and (min-width: 872px) {
+  * {
+    font-family: "Inter", sans-serif;
+  }
+
```

Comment:
- [ ] Good job on implementing some of the previous reviewer's feedback, but still, you are not figuring out some issues. Make sure when using a mobile-first approach, screen size is designed based on mobile-first, so for the media query, we use `min-width` :100:  If you design for a large-screen, please the media query for the desktop version comes at the end of your CSS, which was used for the mobile version; behind that, the desktop version overrides the mobile version. Here, in this case, the mobile-first approach disappears because of using media query at the beginning  Kindly fix the issue :wrench: . For more details, [refer to this ](https://www.w3schools.com/css/css3_mediaqueries.asp)

 - ***Desktop version***

| Figma design | Your design |
|-------------------|-----------------|
| ![Screenshot from 2022-11-04 15-26-27](https://user-images.githubusercontent.com/65129228/199972611-d504418f-f73f-4221-876a-a6b18658f89b.png) | ![Screenshot from 2022-11-04 15-04-08](https://user-images.githubusercontent.com/65129228/199972695-343315ec-1f3d-4152-a4be-7b47075be0cb.png) |
| ![Screenshot from 2022-11-04 15-26-59](https://user-images.githubusercontent.com/65129228/199972851-6b0f79a4-9372-4253-a424-e344e3b0f18c.png) | ![Screenshot from 2022-11-04 15-04-50](https://user-images.githubusercontent.com/65129228/199973189-bf5a1576-1cff-47db-bd87-3138029f7bea.png) |






- ***Your Mobile design***
![Screenshot from 2022-11-04 15-05-27](https://user-images.githubusercontent.com/65129228/199973861-edd51139-ef74-46ac-be91-3976115b4c2d.png)





Example

- Large mobile screen

```
@media only screen and (min-width: 500px) and (max-width: 767px) {
  .headline-content p {
    max-width: 60%;
  }

```
- Medium size screen

```
@media only screen and (min-width: 768px) and (max-width: 999px) {
  .headline-content h1 {
    font-size: 3.625rem;
  }

```
- Desktop and large screen size

```
@media only screen and (min-width: 1000px) {
   .headline-content h1 {
    font-size: 3.825rem;
  }

```